qiskit/circuit/library/generalized_gates/rv.py

```diff
@@ -0,0 +1,77 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2020
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Rotation around an arbitrary axis on the Bloch sphere."""
+
+import numpy
+from qiskit.circuit.gate import Gate
+
+
+class RVGate(Gate):
+    r"""Rotation around arbitrary rotation axis v where |v| is
+    angle of rotation in radians.
+
+    **Circuit symbol:**
+
+    .. parsed-literal::
+
+             ┌──────┐
+        q_0: ┤ R(v) ├
```

Comment:
It currently draws as 
```
     ┌───────────┐
q_0: ┤ RV(x,y,z) ├
     └───────────┘
```
could you update it to show as `R`?

Reply:
The RGate already uses `R` which may cause confusion if this were to have the same symbol and only distinguished itself by the number of arguments.

Reply:
True, though I think `R` makes more sense to be a general rotation -- maybe the current `R` should be renamed to reflect that its a rotation in the x-y plane.

Reply:
It would involve a period of deprecation for the current RGate.